### PR TITLE
matt-roeckers adds Reedsburg, WI file

### LINKED ISSUE #4: matt-roeckers practices using GitHub Flow
- [x] create a branch off of master
- [x] add some commits
- [x] open a pull request
- [x] collaborate :star: 
- [x] merge when we get at least 1 :+1:

### PR DESCRIPTION
**Pull request by matt-roeckers**
The purpose of this PR is to add Reedsburg, WI file to the hometown files.

This PR closes #4

### CODE REVIEW

File: matt-roeckers-reedsburg.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Reedsburg, WI
+
+Population
+- 9,533
```
Comment (scott-larson) [inline matt-roeckers-reedsburg.txt]: Is that more or less than when you lived there?

File: matt-roeckers-reedsburg.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+Mascot
+- Beaver
```
Comment (scott-larson) [inline matt-roeckers-reedsburg.txt:7]: What were your school colors?

**Comment by matt-roeckers** [comment]
@scott-larson responded to your questions in 99837ed

**Review by scott-larson — APPROVED** [review]
Looks good.

**Pull request merged**